### DISCUSSION TITLE
The application for the GitHub Student Developer Pack always fails.

### DISCUSSION THREAD

**Discussion by ovws** [community/community#146452] [GitHub Education]
### Hi everyone,

The information I filled in is all true, but I still can't pass the verification. Meanwhile, your company requires my identity information to match my billing information. My name is "齐文崧". Please tell me how can my surname be considered too short?

![2024-12-06_173929_276.png](https://img.picui.cn/free/2024/12/06/6752c863538cf.png)

**Comment by ovws** [comment]
who can help me ?

**Comment by V2U-Services** [comment]
Verify that you are adhering to every rule specified in the GitHub Student Developer Pack application. Verify your student status again using a legitimate ISIC card or school email. Verify that your college or university is acknowledged. Get help from GitHub support by providing specific error details if the problem continues.

**Comment by ovws** [comment]
未处理。
